saraho1123/currency-exchange
# Testing/unit & integration tests

## Is this a fix or a feature?
testing

## What is the change?
+ Added unit tests for all components
+ Add an integration test to App
+ Made minor changes to Form component

## Where should the reviewer start?
line 1 of the test file for each of the 6 Components

## How should this be tested?
+ run `npm install`
+ run `npm test`
All tests should be passing (with some warnings). 20 tests in all.

## Relevant Tickets
+ #8 
+ #14
